src/languages/swift.js

```diff
@@ -460,6 +460,31 @@ export default function(hljs) {
     end: /}/
   };
 
+  const CLASS_FUNC_DECLARATION = {
+    begin: [
+      /(class)\s+/,          
+      /(func)\b/,
+      /\s+/,
+      /\b[A-Za-z_][A-Za-z0-9_]*\b/ 
+    ],
+    beginScope: {
```

Comment:
Pretty sure if there is no `end` you can use the simpler:

- `match`
- `scope`